editor/src/components/editor/component-button.tsx

```diff
@@ -0,0 +1,214 @@
+/** @jsx jsx */
```

Comment:
This file contains both the _actual_ button and the popup since it's "attached" to the button. If anyone has a better naming suggestion...